test/parallel/test-repl-stdin-push-null.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+require('../common');
+
+// This test ensures that the repl does not
+// throw when using process.stdin stream to push null
+// Refs: https://github.com/nodejs/node/issues/41330
+
+const r = require('repl').start();
+
+// Should not throw.
+r.write('process.stdin.push(null)\n');
+r.write('.exit\n');
```

Comment:
This test already passes with current version of Node.js, that means it's failing to replicate the issue at hand.
https://github.com/nodejs/node/blob/406e6d87c4375037ca52abb5750a3d847f03bc21/doc/guides/collaborator-guide.md#L204-L205